platform/linux-generic/pktio/dpdk.c

```diff
@@ -638,10 +638,16 @@ static inline int mbuf_to_pkt(pktio_entry_t *pktio_entry,
 		pkt_hdr = packet_hdr(pkt);
 
 		if (layer) {
+			int ret;
+
 			packet_parse_reset(pkt_hdr, 1);
-			if (_odp_dpdk_packet_parse_common(pkt_hdr, data,
-							  pkt_len, pkt_len, mbuf,
-							  layer, pktin_cfg) < 0) {
+			ret = _odp_dpdk_packet_parse_common(pkt_hdr, data, pkt_len, pkt_len,
+							    mbuf, layer, pktin_cfg);
+			if (ret)
+				odp_atomic_inc_u64(&pktio_entry->s
+						   .stats_extra.in_errors);
```

Comment:
Should fit into one line. Same comment for all `odp_atomic_inc_u64()` counter increment changes.

Reply:
v5.